src/__experimental__/components/radio-button/radio-button-group.component.js

```diff
@@ -1,15 +1,39 @@
 import React from "react";
 import PropTypes from "prop-types";
 
+import styledSystemPropTypes from "@styled-system/prop-types";
```

Comment:
Minor but this should be grouped with the external imports with the line break after it really